tools/infrasys/src/root.rs

```diff
@@ -48,22 +60,26 @@ pub fn create_root(root_role_path: &Path) -> Result<()> {
         thing: "root role",
     })?;
     fs::create_dir_all(role_dir).context(error::MkdirSnafu { path: role_dir })?;
+
+    let default_region = thread::spawn(move || {
+        let runtime = tokio::runtime::Runtime::new().context(error::RuntimeCreationSnafu)?;
+        runtime.block_on(async_get_region())
+    })
+    .join()
+    .map_err(|_| error::Error::ThreadJoin {})??;
```

Comment:
We should do this the same way as `pubsys`, with the tokio runtime that drops out of scope before interacting with `tough`.

Reply:
fixed